Dockerfile

```
@@ -11,7 +11,10 @@ WORKDIR /build
 
 COPY . /build/
 
-RUN python setup.py install
+RUN apt-get update && \
+    apt-get install -y gcc libc6-dev && \
+    rm -rf /var/lib/apt/lists/* && \
+    python setup.py install
```

I would write apt and python setup steps in separated layers:

```suggestion
    rm -rf /var/lib/apt/lists/*
    
RUN python setup.py install
```

---

Makes sense, thanks)